lms/djangoapps/courseware/tabs.py

```diff
@@ -44,9 +44,9 @@ class CoursewareTab(EnrolledTab):
     def __init__(self, tab_dict):
         def link_func(course, reverse_func):
             if course_home_mfe_outline_tab_is_active(course.id):
-                return get_microfrontend_url(course_key=course.id, view_name=self.view_name)
+                return get_microfrontend_url(course_key=course.id, view_name='home')
```

Comment:
Correcting the URL to return to the Outline Tab once this flag is active in prod.